packages/snap-store-mobx/src/Autocomplete/Stores/FacetStore.test.ts

```diff
@@ -43,7 +43,7 @@ describe('Facet store', () => {
 	});
 
 	it('adds a preview function to each facet value', () => {
-		const searchData = mockData.autocompleteMeta();
+		const searchData = mockData.searchMeta();
```

Comment:
Since this is in Autocomplete store, we should probably use autocomplete responses. This may mean you need to add a new data mock.